roles/remove-node/post-remove/tasks/main.yml

```diff
@@ -2,8 +2,10 @@
 - name: remove-node | Delete node
   command: "{{ kubectl }} delete node {{ kube_override_hostname|default(inventory_hostname) }}"
   delegate_to: "{{ groups['kube_control_plane']|first }}"
-  # ignore servers that are not nodes
-  when: inventory_hostname in groups['k8s_cluster'] and kube_override_hostname|default(inventory_hostname) in nodes.stdout_lines
+  when:
+    - groups['kube_control_plane'] | length > 0
```

Comment:
When is the kube_control_plane group not greater than 0?

Reply:
There is a case: where you want to delete a node worker created and provisioned from the dynamic inventory, but the kubernetes cluster has not been configured yet, along with `kube_control_node`. This is a real problem for me.